### PR TITLE
feat: add tryItOutEnabled configuration

### PR DESCRIPTION
**Pull request by kyle-apex**
allow users to set tryItOutEnabled: true to display
the "Try it out" section by default

### Motivation and Context
Addressed #2805 

### How Has This Been Tested?
Ran in browser by setting ?tryItOutEnabled=true
Added Cypress test.

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [X] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [X] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [X] My changes require a change to the project documentation.
- [X] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [X] My changes can and should be tested by unit and/or integration tests.
- [X] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by mathis-m** [comment]
Fixes #6528

**Review by tim-lai — CHANGES_REQUESTED** [review]
@kyle-apex Thanks for the PR! Functionally looks good, but let's add a "negative" test, where `?tryItOutEnabled=false` still renders properly and doesn't display an expanded view.

**Comment by kyle-apex** [comment]
@tim-lai Thanks for the tip.  As it turns out, setting tryItOutEnabled=false behaved as if it was true.  I fixed the issue for this configuration; however, it's worth mentioning as a future to-do that the previously existing boolean parameter configurations (ex: displayRequestDuration, displayOperationId) would fail the ?parameter=false test.

File: src/core/containers/OperationContainer.jsx [hunk 1]
```diff
@@ -7,8 +7,11 @@ import { Iterable, fromJS, Map } from "immutable"
 export default class OperationContainer extends PureComponent {
   constructor(props, context) {
     super(props, context)
+
+    const { tryItOutEnabled } = props.getConfigs()
+
     this.state = {
-      tryItOutEnabled: false,
+      tryItOutEnabled: tryItOutEnabled && tryItOutEnabled !== "false",
```
Comment (tim-lai) [inline src/core/containers/OperationContainer.jsx:14]:
```suggestion
      tryItOutEnabled: tryItOutEnabled && tryItOutEnabled === "true",
```

**Comment by tim-lai** [comment]
@kyle-apex Thanks, good to know about other cases for future PR(s). I made a commit suggestion, to explicitly check for "true".

**Review by tim-lai — APPROVED** [review]

**Comment by tim-lai** [comment]
@kyle-apex PR merged! Thanks for the contribution!

**Comment by kyle-apex** [comment]
@tim-lai Thanks for your help and patience on my first submission!

**Comment by OndraZizka** [comment]
Hi, when is this going to be released? And will it be controlled only in URL? Or can it be propagated from Spring config YAML, I guess through OpenAPI?

**Comment by mathis-m** [comment]
@OndraZizka I think it can be configured via all available config options.

**Comment by jonatan-bengtsson** [comment]
Hi, I'm using the latest `openapi-ui` in a Spring project. 
Could someone help me figure out how to enable this flag?
It doesn't seem to be a available flag via `springdoc` config in `application.yaml` (yet anyways).
Is there a way to still enable `swagger-ui` config flags?

**Pull request merged**